cpp/cmake/Modules/ccache.cmake

```diff
@@ -0,0 +1,16 @@
+# Use ccache if possible.
```

Comment:
This file needs the NVIDIA copyright at the top of it

Reply:
Done.